### PR TITLE
Replace `nan`s and `inf`s in the reproducer.

### PR DESCRIPTION
**Pull request by wujingyue**

### CODE REVIEW

File: nvfuser/__init__.py [hunk 1]
```diff
@@ -162,7 +163,18 @@ def execute(
                             f".as_strided({tuple(i.size())}, {tuple(i.stride())}),\n"
                         )
                 else:
-                    msg += f"    {i},\n"
+                    input_as_string = str(i)
```
Comment (jjsjann123) [inline nvfuser/__init__.py:166]: I'm wondering how this hasn't shown up in our test with extreme values yet.

nitpick, can we add a test case to verify this?

**Review by jjsjann123 — APPROVED** [review]

File: nvfuser/__init__.py (hunk 1 above)
Comment (wujingyue) [reply nvfuser/__init__.py:166]: > I'm wondering how this hasn't shown up in our test with extreme values yet.

I thought we never auto-test reproducers generated here? 

> nitpick, can we add a test case to verify this?

Maybe. Something like 
1. run a known bad program which generates a reproducer
2. extract the reproducer from stdout
3. run the reproducer to see if it gives the same error
?

Comment (jjsjann123) [reply nvfuser/__init__.py:166]: oh nvm. this is patching the repro scripts. I wasn't paying enough attention here and thought it's the stringification thing from fusion definition. My bad.

Comment (jjsjann123) [reply nvfuser/__init__.py:166]: no need to add test here. Let's merge it.

**Review by jjsjann123 — APPROVED** [review]
Do I need to stamp again?

**Pull request merged**